### DISCUSSION TITLE
Comments get removed from docker compose file

### DISCUSSION THREAD

**Discussion by Conkuist** [louislam/dockge#478] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I like to add some comments to my docker compose files but dockge seems to remove them.
Is there a way to disable this behavior?

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Debian 12

### 🌐 Browser

Firefox 124.0.2

### 🐋 Docker Version

26.0.0

### 🟩 NodeJS Version

v18.17.1

**Comment by trythatagain** [comment]
I've just checked and tried adding a comment and it was there after I did a save.
Can you say how you normally add comments?

**Comment by mikansontap123** [comment]
I'm facing the same issue, and am running all the same OS, browser, and software component versions as @Conkuist. In my case, its not 100% of the time. Sometimes comments are removed, and other times not. I've yet to figure out how to reliably re-produce the issue 100% of the time, but will update here if I do.

**Comment by andersmmg** [comment]
Issue #176 is related, and in my experience it does keep some comments but not others.
